### PR TITLE
feat: Document include_paths DOCS-699

### PR DESCRIPTION
**Pull request by nicklem**
Documents the new field `include_paths` for the Codacy configuration file.

### :eyes: Live preview
https://docs-699-document-new-field-in-coda--docs-codacy.netlify.app/repositories-configure/codacy-configuration-file/

### CODE REVIEW

File: docs/repositories-configure/codacy-configuration-file.md [hunk 1]
```diff
@@ -92,6 +94,30 @@ exclude_paths:
   - "**/*.resource"
 ```
 
+## Including specific files using a Codacy configuration file {: id="include-files"}
+
+The Codacy configuration file allows you to explicitly specify files or directories to include in the analysis. This is particularly useful for [bypassing files or directories that are ignored by default](./ignoring-files.md#default-ignored-files) or specified in `exclude_paths`.
+
+!!! note
+    If both `exclude_paths` and `include_paths` are defined, `include_paths` will specify exceptions to the exclusions defined in `exclude_paths`.
```
Comment (nicklem) [inline docs/repositories-configure/codacy-configuration-file.md:102]: @codacy-vrhpires Can you confirm this is the case? Thanks

Comment (codacy-vrhpires) [reply docs/repositories-configure/codacy-configuration-file.md:102]: @nicklem It is.

**Comment by nicklem** [comment]
@codacy-vrhpires Something like [this](https://github.com/codacy/docs/pull/2130/commits/8d5e13c8d568bcc369c3bc040ca6b578d3889015)?

**Review by codacy-vrhpires — APPROVED** [review]

File: docs/repositories-configure/codacy-configuration-file.md [hunk 2]
```diff
@@ -92,6 +98,30 @@ exclude_paths:
   - "**/*.resource"
 ```
 
+## Including specific files using a Codacy configuration file {: id="include-files"}
+
+The Codacy configuration file allows you to explicitly specify files or directories to include in the analysis. This is particularly useful for [bypassing files or directories that are ignored by default](./ignoring-files.md#default-ignored-files) or specified in `exclude_paths`.
```
Comment (claudiacarpinteiro) [inline docs/repositories-configure/codacy-configuration-file.md:103]: Maybe we can add a note/tip in the [Default ignored files](https://github.com/codacy/docs/blob/DOCS-699-document-new-field-in-codacy-config-file-include-paths/docs/repositories-configure/ignoring-files.md?plain=1#L29) section about this bypassing possibility? 🤔

**Review by claudiacarpinteiro — APPROVED** [review]
@nicklem I left a small comment, but already looks good to go! 🚀

File: docs/repositories-configure/codacy-configuration-file.md (hunk 2 above)
Comment (nicklem) [reply docs/repositories-configure/codacy-configuration-file.md:103]: Good idea. Done 👍

**Review by codacy-vrhpires — APPROVED** [review]

**Pull request merged**